hokusai/commands/console.py

```diff
@@ -65,7 +65,7 @@ def console(context, shell, tag, with_config, with_secrets, env):
     job_id = k8s_uuid()
 
   try:
-    call(verbose("kubectl run %s-shell-%s -t -i --image=%s:%s --restart=OnFailure --rm %s -- %s" %
+    call(verbose("kubectl run %s-shell-%s -t -i --image=%s:%s --image-pull-policy=Always --restart=OnFailure --rm %s -- %s" %
```

Comment:
i think by default we have `imagePullPolicy: Always` in our `staging.yml` does this ignore that field?